a3p-integration/proposals/a:vaults-auctions/prepare.sh

```diff
@@ -4,3 +4,5 @@
 # actions are executed in the upgraded chain software and the effects are
 # persisted in the generated image for the upgrade, so they can be used in
 # later steps, such as the "test" step, or further proposal layers.
+
```

Comment:
another reason to remove this file is it says it's for after the proposal is executed but you're running it as part of the eval